ui/components/popover/popover.scss

```diff
@@ -4,6 +4,9 @@ button.popoverButton {
   border: 1px solid gray;
   border-radius: 100%;
   margin: 0;
+  &.smallerPopoverButton {
+    padding: 2px 6px !important;
```

Comment:
We have a lot of global styles that are hard to override due to depth of specificity.
Ideally, we whittle those down and reduce the need for `!important`